proto/cronos/cronos.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package cryptoorgchain.cronos.cronos;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/crypto-org-chain/cronos/x/cronos/types";
+
+// Params defines the parameters for the cronos module.
+message Params {
+    option (gogoproto.goproto_stringer)             = false;
+    repeated ConvertEnabled convert_enabled         = 1 [(gogoproto.moretags) = "yaml:\"convert_enabled,omitempty\""];
+    string                  ibc_cro_denom           = 2 [(gogoproto.moretags) = "yaml:\"ibc_cro_denom,omitempty\""];
+    string                  ibc_cro_channelID       = 3 [(gogoproto.moretags) = "yaml:\"ibc_cro_channelID,omitempty\""];
+}
+
+// ConvertEnabled maps coin denom to a convert_enabled status (whether a denom is
+// convertable).
+message ConvertEnabled {
+    option (gogoproto.equal)            = true;
+    option (gogoproto.goproto_stringer) = false;
+    string denom                        = 1;
+    bool   enabled                      = 2;
```

Comment:
Is this field necessary? can we remove the denom from the list when disabling?

Reply:
yes, in the future, we would have the mapping between denom <-> config (smart contract address, ibc config etc)

enable = false means that it is supported but not enable at the moment

Reply:
I see, if you plan to do ibc tokens coversion in another PR, maybe we can focus on CRO itself in this PR, to avoid some confusion, and easier to review I think.